frontend/src/ui/Video.tsx

```diff
@@ -9,6 +9,8 @@ type ThumbnailProps = JSX.IntrinsicElements["div"] & {
         title: string;
         thumbnail: string | null;
         duration: number;
+        isLive: boolean;
+        created: string;
```

Comment:
I think I would have this accept a `Date` already :thinking:

Reply:
While I would like that as well, this would mean doing the `new Date` conversion at all 4 call sites, two of which look like `<Thumbnail event={event}`. So changing this would add lot more code. 

Of course, the best option would be for relay to offer an option for `created` to be a `Date`, but I don't think that works (I tried for a bit). But yeah, so I would leave it as is?

Reply:
Ah ye ok, sure. :shrug: